tools/debian/rules

```diff
@@ -5,11 +5,7 @@ DEB_HOST_MULTIARCH ?= $(shell dpkg-architecture -qDEB_HOST_MULTIARCH)
 # we need an apparmor profile for >= 4.0; this exists in Ubuntu >= 24.04, but not in Debian yet
 PRE4AA = $(filter $(shell . /etc/os-release; echo $${VERSION_ID:-unstable}),22.04 11 12 unstable)
 
-# pcp isn't in Debian testing right now
-ifeq ($(shell . /etc/os-release; echo $${VERSION_CODENAME}),trixie)
-    export DH_OPTIONS = -Ncockpit-pcp
-    CONFIG_OPTIONS := --disable-pcp
-endif
+CONFIG_OPTIONS := --disable-pcp
```

Comment:
This is weird -- that option will go away entirely, and this PR should at least make that the default? Can you just leave it empty?

Reply:
WDYM? It doesn't seem to make sense to make disable-pcp enable-pcp in this PR? I thought we discussed that, if that needs to happen I can ofcourse make that happen.

Reply:
Not sure what you mean with "make disable-pcp enable-pcp". Oh, is that because we still have all the autoconfery and the source code in main? Then it's fine of course.

Reply:
It is, PCP is enabled by default.